main.py

```diff
@@ -0,0 +1,105 @@
+from typedb.client import *
+
+def highestVaccines():
+
+    with TypeDB.core_client("localhost:1729") as client:
+        with client.session("vaccination_schema", SessionType.DATA) as session:

```

Comment:
I changed the name of the database to 'vaccination'.
But I am unable to detect the URL on which TypeDB is running, instead of localhost:1729.
Can you please guide a bit on that?

Reply:
I wouldn't expect you to be able to detect the URL - the user could be running it anywhere on the planet with conceivably any IP address or URL - the solution would more likely be taking the URL (and potentially database name) from the user through either a command line argument or using `input()` at the start of the program

Reply:
Took input from the user in the beginning of the program.